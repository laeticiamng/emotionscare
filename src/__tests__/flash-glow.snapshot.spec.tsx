import { describe, it, expect } from "vitest";
import { render } from "@testing-library/react";
import Page from "@/app/modules/flash-glow/page";

describe("FlashGlowPage", () => {
  it("rend la page", () => {
    const { container } = render(<Page />);
    expect(container).toMatchSnapshot();
  });
});
