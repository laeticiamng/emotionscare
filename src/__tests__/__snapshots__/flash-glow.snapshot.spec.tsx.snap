// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FlashGlowPage > rend la page 1`] = `
<div>
  <main
    class="p-4 space-y-4"
  >
    <div>
      Flash Glow
    </div>
    <p>
      Bienvenue dans le module Flash Glow.
    </p>
    <button>
      Commencer
    </button>
  </main>
</div>
`;
